src/index.js

```diff
@@ -45,7 +45,7 @@ const printString = (str) => {
       .replace(/\\\n\s*/g, "")
       .split(/(\s)/)
       .map((s) =>
-        s === " " ? ifBreak(group([" ", "\\", hardline, " "]), " ") : s,
+        s === " " ? line : s,
```

Comment:
This will create issues:
```diff
 let settings.autocue.internal.sustained_endings_dropoff =
   settings.make(
-    description=
-      "Max. loudness drop off immediately after crossfade point to consider it \
-       as relevant ending [percentage]",
+    description="Max. loudness drop off immediately after crossfade point to
+    consider it as relevant ending [percentage]",
     15.0
   )
```

Reply:
Oops! Restored it. Didn't add group back around it, since that doesn't change the behavior. The breaking decision is made at the fill level.